carsus/io/cmfgen/base.py

```diff
@@ -430,12 +433,12 @@ def load(self, fname):
         self.meta = meta
 
     def to_hdf(self, key='/photoionization_cross_sections'):
+        if len(self.base) > 0:
+            with pd.HDFStore(self.fname + '.h5', 'a') as f:
 
-        with pd.HDFStore(self.fname + '.h5', 'a') as f:
-            header = pd.Series(data=self.meta)  # FIXME: couldn't write this like `attr` metadata
-            f.put(key, header, format='table', data_columns=True)
+                for i in range(1, len(self.base)-1):
+                    subkey = key + '/' + str(i)
```

Comment:
use format  here. it's also smaller `'{0}/{1}'.format(key, i)`

Reply:
I don't know why I only use format for print statements and not for strings in general :/